### DISCUSSION TITLE
Popover full width

### DISCUSSION THREAD

**Discussion by Skuallpa** [element-plus/element-plus#13394] [Q&A]
I'd like to center an `el-popover` with a desired with on screen, so that it display nicely centered on mobile, but have been unable to do so. 

I've tried to compute the size dynamically using `window.innerWidth` (minus a little margin) but it doesn't display centered (horizontally). 

![Screenshot 2023-06-28 at 09 41 02](https://github.com/element-plus/element-plus/assets/8036638/73220248-fce2-4379-8863-6e9a8c1a3125)

I've investigate with the `modifiers` of popper.js using the `popper-options` attribute of the `el-popover`, but couldn't find a way to make it display perfectly in the center.
https://popper.js.org/docs/v2/modifiers/

Any idea?

**Comment by hibichann** [comment]
[playground](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6Ijx0ZW1wbGF0ZT5cbjxkaXYgc3R5bGU9XCJkaXNwbGF5OmZsZXg7anVzdGlmeS1jb250ZW50OiBzcGFjZS1hcm91bmQ7XCI+XG4gIDxlbC1wb3BvdmVyXG4gICAgcGxhY2VtZW50PVwiYm90dG9tXCJcbiAgICA6d2lkdGg9XCIyMDBcIlxuICAgIHRyaWdnZXI9XCJob3ZlclwiXG4gID5cbiAgPGRpdiBzdHlsZT1cInRleHQtYWxpZ246IGNlbnRlcjtcIj48c3Bhbj4xMjM8L3NwYW4+PC9kaXY+XG4gICAgPHRlbXBsYXRlICNyZWZlcmVuY2U+XG4gICAgICA8YnV0dG9uPjEyMzwvYnV0dG9uPlxuICAgIDwvdGVtcGxhdGU+XG4gIDwvZWwtcG9wb3Zlcj5cbiAgPC9kaXY+XG48L3RlbXBsYXRlPiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwidHNjb25maWcuanNvbiI6IntcbiAgXCJjb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlXCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVSZXNvbHV0aW9uXCI6IFwiQnVuZGxlclwiLFxuICAgIFwiYWxsb3dJbXBvcnRpbmdUc0V4dGVuc2lvbnNcIjogdHJ1ZVxuICB9LFxuICBcInZ1ZUNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogMy4zXG4gIH1cbn1cbiIsIl9vIjp7fX0=)

you may just need another div to wrap it or change placement to `bottom` or just use a default slot? im not sure if i understand your question.

**Reply by Skuallpa** [reply]
Thanks for your reply. I'm not talking about centering the content of the popover, but the popover itself, like this:
<img width="287" alt="Screenshot 2023-09-10 at 09 12 07" src="https://github.com/element-plus/element-plus/assets/8036638/efba19dc-b89b-4194-a65e-c25f1aafe47b">
